### PR TITLE
Restructure main admin menu

### LINKED ISSUE #2586: Re-design Menus
- New Icon (should be base64 SVG icon so it's compatible with Calypso)
- Should have all the current same submenu items (Except Add New, remove those)
- Add "Applications pro" and "Resumes pro" items
- The Pro upsell items will link to pages that have more information on that add-on
- There will be a separate task for creating those pages.

I'm not sure if we will rename our "Add-ons" page to "Extensions" so I asked here: p6r3EZ-1Nu-p2#comment-3358
Decision:  Will be called "Extensions"

Related: https://github.com/Automattic/wpjm-addons/issues/118

Design:
Image

### PR DESCRIPTION
**Pull request by fjorgemota**
Fixes #2586 

### Changes Proposed in this Pull Request

- Restructure menu to match the design as shown on #2586 

### Testing Instructions

Using this branch of the plugin to test:

1. Make sure the menu matches the design
2. Check the icon, order of the items, and labels used in the menu
3. Verify if no icons break with the changes

### Release Notes

* Restructure the main admin menu of the plugin

### Screenshot / Video

New restructured menu

----

| Plugin build for 941dfb842a4431a09b7ddfa2be6ed935a9339d3d <a href="#"></a> |
| ------------------------------------------------------------ |
| 📦 [Download plugin zip](https://wpjobmanager.com/wp-content/uploads/2023/10/wp-job-manager-zip-2612-941dfb84.zip)                       |
| ▶️ [Open in playground](https://wpjobmanager.com/playground/?core=2023/10/2612-941dfb84)             |

### CODE REVIEW

File: assets/css/icons.scss [hunk 1]
```diff
@@ -1,23 +1,12 @@
...
-	src:
-		url(../font/jm-logo/jm.eot?#iefixycsbky) format("embedded-opentype"),
-		url(../font/jm-logo/jm.woff?ycsbky) format("woff"),
-		url(../font/jm-logo/jm.ttf?ycsbky) format("truetype"),
-		url(../font/jm-logo/jm.svg?ycsbky#icomoon) format("svg");
+		url(../font/job-manager.eot?16367094#iefix) format("embedded-opentype"),
+		url(../font/job-manager.woff2?16367094) format('woff2'),
+		url(../font/job-manager.woff?16367094) format("woff"),
+		url(../font/job-manager.ttf?16367094) format("truetype"),
+		url(../font/job-manager.svg?16367094#job-manager) format("svg");
```
Comment (yscik) [inline assets/css/icons.scss]: There is no need for the extra formats nowadays, `woff2` works everywhere:
https://developer.mozilla.org/en-US/docs/Web/CSS/@font-face#browser_compatibility

File: includes/admin/class-wp-job-manager-admin.php [hunk 2]
```diff
@@ -166,10 +166,14 @@ public function admin_enqueue_scripts() {
...
 	public function admin_menu() {
+		remove_submenu_page( 'edit.php?post_type=job_listing', 'post-new.php?post_type=job_listing' );
+		$item = remove_submenu_page( 'edit.php?post_type=job_listing', 'edit.php?post_type=job_listing' );
+		// change item label to "Job Listings".
+		add_submenu_page( 'edit.php?post_type=job_listing', $item[0], __( 'Job Listings', 'wp-job-manager' ), $item[1], $item[2], '', 0 );
 		add_submenu_page( 'edit.php?post_type=job_listing', __( 'Settings', 'wp-job-manager' ), __( 'Settings', 'wp-job-manager' ), 'manage_options', 'job-manager-settings', [ $this->settings_page, 'output' ] );
 
 		if ( WP_Job_Manager_Helper::instance()->has_licensed_products() || apply_filters( 'job_manager_show_addons_page', true ) ) {
-			add_submenu_page( 'edit.php?post_type=job_listing', __( 'WP Job Manager Add-ons', 'wp-job-manager' ), __( 'Add-ons', 'wp-job-manager' ), 'manage_options', 'job-manager-addons', [ $this, 'addons_page' ] );
+			add_submenu_page( 'edit.php?post_type=job_listing', __( 'WP Job Manager Extensions', 'wp-job-manager' ), __( 'Extensions', 'wp-job-manager' ), 'manage_options', 'job-manager-addons', [ $this, 'addons_page' ] );
```
Comment (yscik) [inline includes/admin/class-wp-job-manager-admin.php]: I think the latest is that we are calling this 'Marketplace': p6r3EZ-1Nu-p2#comment-3379

File: includes/class-wp-job-manager-post-types.php [hunk 3]
```diff
@@ -247,7 +247,7 @@ public function register_post_types() {
 						'labels'               => [
 							'name'              => $plural,
 							'singular_name'     => $singular,
-							'menu_name'         => ucwords( $plural ),
+							'menu_name'         => __( 'Types', 'wp-job-manager' ),
```
Comment (yscik) [inline includes/class-wp-job-manager-post-types.php:250]: Should we also rename 'Job Categories' to 'Categories'? 
image

**Review by yscik — APPROVED** [review]
Looks good!

File: assets/css/icons.scss (hunk 1 above)
Comment (fjorgemota) [reply assets/css/icons.scss]: Good catch! Removed in 770ac8f. Thanks!

File: includes/admin/class-wp-job-manager-admin.php (hunk 2 above)
Comment (fjorgemota) [reply includes/admin/class-wp-job-manager-admin.php]: Good catch! Renamed in a79e326 and 941dfb8

File: includes/class-wp-job-manager-post-types.php (hunk 3 above)
Comment (fjorgemota) [reply includes/class-wp-job-manager-post-types.php:250]: This is a good question. I renamed in 2ccdcab, but then we might want to update the Job Tags addon to use only "Tags" on the menu? Not sure.

**Pull request merged**